### DISCUSSION TITLE
List Bookmarking Triggers 'Contributor' Section [iOS]

### DISCUSSION THREAD

**Discussion by haleygelato** [community/community#114801] [Mobile]
### Select Topic Area

Bug

### Body

On iOS, bookmarking a list would also trigger the "contributor" section. It would go from being triggered to untriggered without an increase in the number of contributors.

https://github.com/community/community/assets/112786962/1f3c5255-db17-472a-bf13-d2b500a291e5

**Comment by Iron-Ham** [comment]
Hello @haleygelato,

Thanks for reaching out. The behavior that you're describing is as intended: the contributor count shouldn't be increasing as a result of having starred a list. Rather, the starred-count should be increasing. In the video posted above, the contributor label shows a heart – indicating that you've starred the repository, and the star-label shifts color and increases the count.

I understand that this may not be as clear as it should be. What behavior were you expecting instead, and is the visual representation of tapping the "Star" button confusing? 

Thanks, 
Hesham

**Reply by haleygelato** [reply]
Thank you!

<img width="302" alt="Screenshot 2024-04-01 at 12 30 42 PM 1" src="https://github.com/community/community/assets/112786962/5de5aa46-113a-4282-a9b6-c0e6896a089e">

As I understand it, 

- Yes the contributor count shouldn't be increasing
- Starring a list should only cause the star section to appear as starred

<img width="69" alt="Screenshot 2024-04-01 at 12 22 58 PM" src="https://github.com/community/community/assets/112786962/82d3a647-2928-4061-8b18-05f829672529">

- However, starring a list will also cause the person icon to turn into a red heart, and the subsequent section with the contributor count will also turn red. Then it will revert back to the person icon and its original color. This looks odd because starring a list shouldn't affect the contributor list at all, which is why I initially thought it was a bug. Especially with the star icon in the star section will also simultaneously transition from unstarred to starred.

<img width="139" alt="Screenshot 2024-04-01 at 12 29 53 PM" src="https://github.com/community/community/assets/112786962/4625515d-8ca3-4e6c-a8ae-9823b785e528">
<img width="161" alt="Screenshot 2024-04-01 at 12 24 03 PM" src="https://github.com/community/community/assets/112786962/10832d23-35e6-47f9-af4b-28a2c1d6b36b">
